### DISCUSSION TITLE
Welcome to Dotfiles Discussions!

### DISCUSSION THREAD

**Discussion by 2KAbhishek** [2KAbhishek/dots2k#10] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by TheCollinsByte** [comment]
Name: Collin
Location: East Africa
About Me: Learner, Intruder etc
[Github Account](https://github.com/paradigm-lab)
Looking forward to learn and share ideas with the community. Let build great paradigms
